### ISSUE TITLE
List available endpoints instead of error message

### ISSUE THREAD

**Issue by orkunkl** [iov-one/bns#9]
Currently, if you browse to the bnsapi without specifying an endpoint you see

```json
{
	"errors": [
		"Not Found"
	]
}
```

We need a structured endpoint listing such as cosmos:

```
Available endpoints:
Endpoints that require arguments:
//63.35.133.166/abci_info?
//63.35.133.166/abci_query?path=_&data=_&height=_&prove=_
//63.35.133.166/block?height=_
//63.35.133.166/block_results?height=_
//63.35.133.166/blockchain?minHeight=_&maxHeight=_
//63.35.133.166/broadcast_evidence?evidence=_
//63.35.133.166/broadcast_tx_async?tx=_
//63.35.133.166/broadcast_tx_commit?tx=_
//63.35.133.166/broadcast_tx_sync?tx=_
//63.35.133.166/commit?height=_
//63.35.133.166/consensus_params?height=_
//63.35.133.166/consensus_state?
//63.35.133.166/dump_consensus_state?
//63.35.133.166/genesis?
//63.35.133.166/health?
//63.35.133.166/net_info?
//63.35.133.166/num_unconfirmed_txs?
//63.35.133.166/status?
//63.35.133.166/subscribe?query=_
//63.35.133.166/tx?hash=_&prove=_
//63.35.133.166/tx_search?query=_&prove=_&page=_&per_page=_
//63.35.133.166/unconfirmed_txs?limit=_
//63.35.133.166/unsubscribe?query=_
//63.35.133.166/unsubscribe_all?
//63.35.133.166/validators?height=_
```

Acceptance:
- [x] Endpoints with and w/o parameters must be shown separately
- [ ] Listed endpoints should be links, not just plain text

**Comment by davepuchyr** [comment]
@orkunkl, I edited the acceptance criteria and added several comments to my review.  I guess you didn't see them.

- [ ] Listed endpoints should be links, not just plain text
- [ ] Links should be prefixed with `//`
- [ ] `Available endpoints with query parameter` should be `Available endpoints with query parameters`
- [ ] `Available endpoints w/o parameter:` should be `Available endpoints without parameters:`
- [ ] The link for docs should be on a newline, like `//localhost/info/`

**Comment by davepuchyr** [comment]
I didn't see that you merged #13.
